src/error.c

```diff
@@ -42,6 +46,15 @@ stumpless_get_error_id( const struct stumpless_error *err ) {
   return err->id;
 }
 
+const char *
+stumpless_get_error_id_string( enum stumpless_error_id id) {
+  if ( stumpless_error_enum_to_string[id] ) {
+    return stumpless_error_enum_to_string[id];
+  }
+  
+  return "NO_SUCH_ERROR_ID";
```

Comment:
You need to add a test to cover the "NO_SUCH_ERROR_ID" scenario, as it is currently [uncovered](https://codecov.io/gh/goatshriek/stumpless/pull/99/diff) and responsible for the pull request check failure you are seeing. I recommend adding the test before you fix the bounds check, so that you see it segfault before the fix and pass afterwards.